modules/db_instance/outputs.tf

```diff
@@ -76,6 +76,7 @@ output "this_db_instance_name" {
 output "this_db_instance_username" {
   description = "The master username for the database"
   value       = local.this_db_instance_username
+  sensitive   = true
```

Comment:
Do you have any idea why `username` is sensitive? I guess it is sensitive in the AWS provider but why really? What can be the reason for that? It seems counterintuitive to have it this way. :)